src/libstd/rt.rs

```diff
@@ -22,11 +22,13 @@
             issue = "0")]
 #![doc(hidden)]
 
-
-
 // Reexport some of our utilities which are expected by other crates.
 pub use panicking::{begin_panic, begin_panic_fmt, update_panic_count};
 
+// Reexport the start module on platforms that provide it
+#[unstable(feature = "sys_rt", issue="0")]
+pub use sys::rt::*;
```

Comment:
Hygiene error! In order to generate executables, we have to have a naked _start function at the libstd root.